docs/index.md

```diff
@@ -0,0 +1,26 @@
+title: Pantheon overview
```

Comment:
Title and desc should be changed as it's what appear when you link to it. So more something like 
title: Pantheon Enterprise Ethereum client
description: Pantheon is an open-source Enterprise Ethereum client developed under the Apache 2.0 license and written in Java. It runs on the Ethereum public network, private networks, and test networks.